start-dev.sh

```
@@ -0,0 +1,226 @@
+    echo -e "${BLUE}🔧 Starting backend server on port 5001...${NC}"
+    cd backend
+    npm run dev > ../logs/backend.log 2>&1 &
+    BACKEND_PID=$!
+    cd ..
+    echo $BACKEND_PID > .backend.pid
+    echo -e "${GREEN}✅ Backend started (PID: $BACKEND_PID)${NC}"
+}
+
+start_frontend() {
+    echo -e "${BLUE}🎨 Starting frontend server on port 3000...${NC}"
+    cd frontend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)